app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,6 +1,7 @@
-// @ts-ignore
 import * as yup from "yup";
 
+export const IMG_URL =
```

Comment:
What exactly is this image for? Currently it's just an `image` within the `Notes` namespace? So is this an image of a `Note`, the `User` who created the note? The variable name should convey that.